app/__init__.py

```diff
@@ -49,8 +51,7 @@ def create_app(config_class, database_uri=None):
 
     # Set content security policy
     csp = {
-        "default-src": "'self'",
-        "script-src": ["'self'"],
+        "default-src": f"'self' {app.config['FLASKS3_CDN_DOMAIN']}",
```

Comment:
Content-Security-Policy meta-tag allows you to reduce the risk of [XSS](http://en.wikipedia.org/wiki/Cross-site_scripting) attacks by allowing you to define where resources can be loaded from, preventing browsers from loading data from any other locations. This makes it harder for an attacker to inject malicious code into our site. 

However, now we will want to allow the flask webapp to allow resources from FLASKS3_CDN_DOMAIN to be loaded as well as from self (for when FLASKS3_ACTIVE is False)